src/components/common/layout/footer.jsx

```diff
@@ -11,6 +11,7 @@ import { twMerge } from "tailwind-merge";
 import { Button } from "@/components/common";
 import { repositoryLink, sliitFossMainWebsite } from "@/constants";
 import { FossLogoDark, Timekeeper } from "@/icons";
+import { facebook, twitter, instagram, youTube, linkedIn } from "@/constants/index.js";
```

Comment:
We do not need to explicitly mention index.js here, by default an import will resolve to an index file